text/0119-standard-system-metrics.md

```diff
@@ -0,0 +1,146 @@
+# Standard names for system/runtime metrics
```

Comment:
I think we need to be super clear that we're specifying the names of the `instruments`, and the labels for the recordings, not actual metrics, in this proposal.

Reply:
Definitely, but what is the difference exactly? Just looking at [this part of the spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/metrics/api.md#metric-instruments) where it mentions the instrument names, I took it that the instruments are Counter, UpDownCounter, etc. and that a metric uses the instrument to take measurements.

Reply:
When you acquire an Instrument from a Meter, you give it a name. This generally ends up, at the end of the metric pipeline, resulting in metrics with the same name as the instrument. But, they're not the same thing. Most importantly, there is nothing in the API called "Metric", just Instruments.  

See here for the rules for assigning a name to an instrument: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/metrics/api.md#instrument-naming-requirements

Reply:
I also found it confusing that this OTEP references the host metrics receiver, but if I look at the [code](https://github.com/open-telemetry/opentelemetry-collector/blob/master/receiver/hostmetricsreceiver/internal/scraper/cpuscraper/cpu_scraper.go) I don't see instruments being used. Would it make sense to use instruments there?

Reply:
The collector consumes metrics, after being aggregated, not instruments, so I think that still makes sense.

Reply:
(This is somewhat off topic.)

The idea of using instruments in the collector sounds a lot like using the OTel-Go SDK to generate metrics inside the collector, which could then be "exported" from the SDK into a collector processing pipeline. I don't think this will be worth the complexity, however in the discussion about adding a [Statsd receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/290) we are consuming _unaggregated_ metrics and want to feed them into a collector processing pipeline. 

(We have discussed for Statsd how you could use parts of the OTel-Go SDK to implement this aggregation, mainly the `Aggregator` classes are re-usable. However, I'd prefer not to perform this aggregation in the Statsd receiver--instead I'd like to leverage the proposed [Metrics Transform Processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/332).)

Reply:
@jkwatson thank you for explaining. I think my confusion is between the instrument vs. an instance of the instrument. In the Python API right now, it is not this way; we have [`Meter.create_metric() -> Metric`](https://github.com/open-telemetry/opentelemetry-python/blob/8be986cd6c94368ed769149022a7862f0ca47f35/opentelemetry-api/src/opentelemetry/metrics/__init__.py#L338-L347).